glances_api/__init__.py

```diff
@@ -31,6 +32,7 @@ def __init__(
         self.values = None
         self.username = username
         self.password = password
+        self.verify_ssl = verify_ssl if verify_ssl else True
```

Comment:
`glances` has no support for HTTPS out-of-the-box. This is only relevant if one is running `glances` behind a reverse proxy with a self-signed certificate. I guess that not many installation have used this kind of setup over the last six years while the integration was available.

Has `glances` support for HTTPS now?

Reply:
I'm not sure... This was originally made for supporting the current 'verify_ssl' boolean setting as well

Reply:
The conditional seems superfluous as there is a default.

```suggestion
        self.verify_ssl = verify_ssl
```